mlir/test/Dialect/LLVMIR/func.mlir

```diff
@@ -40,6 +40,7 @@ module {
       arg_attrs = [{llvm.noalias}, {}], xxx = {yyy = 42}} : () -> ()
 
   // CHECK: llvm.func @roundtrip1()
+  // CHECK-GENERIC: "llvm.func"{{.*}}@roundtrip1{{.*}}visibility = "private"
```

Comment:
I am a bit surprised the visibility is private here since I would expect the new funcOp argument is an optional attribute and not a default valued attribute? But it seems to pass tests so I have to miss something :).

Reply:
Reverted, CHECK-GENERIC isn't a prefix that is checked, this line was ignored :)
(it was something I was trying earlier on)

Reply:
Ah, I did miss there was another check prefix...